dna/database_to_json.py

```diff
@@ -90,31 +99,45 @@ def get_pipeline_run_info(self, pipeline_run):
         """
         pipeline = pipeline_run["pipeline"]
         pipeline_run_id = pipeline_run["id"]
-        pipeline_id = pipeline["id"]
         simple_pipeline  = self.parse_simpler_pipeline(pipeline)
         if simple_pipeline is None:
             # This pipeline is not useable for the metadataset.
             return None
         problem_type = self.get_problem_type(pipeline_run["problem"])
         if problem_type == "unsupported":
             return None
-        raw_dataset_name = pipeline_run["datasets"][0]["id"]
-        test_accuracy = pipeline_run["run"]["results"]["scores"][0]["value"]
+        test_score = self.get_score(
+            pipeline_run,
+            self.supported_problem_types[problem_type]["d3m_metric_name"]
+        )
+        if test_score is None:
+            # This pipeline run was not scored using the supported metric for this
+            # problem type.
+            return None
+        dataset_id = pipeline_run["datasets"][0]["id"]
         test_predict_time = self.get_time_elapsed(pipeline_run)
-        train_accuracy = 0
-        train_predict_time = 0  # TODO have this find the fit pipeline and get the time
+        # TODO have this find the fit pipeline and get the train score
+        # and train predict time, then include those in the returned dictionary.
```

Comment:
@epeters3 Can you open an issue for this?

Reply:
*For this TODO

Reply:
Actually, I think in D3M the scores are not computed on the training data.

Reply:
I think you're right. I'll open an issue for capturing just the train time, mentioning that that may be an interesting target for metalearning.

Reply:
This has been documented in #208.